README.md

```diff
@@ -224,35 +224,31 @@ the same URL as production.
 
 For example:
 
-```
-Replay.localhost "www.example.com"
+```javascript
+Replay.localhost('www.example.com');
 ```
 
-There may be hosts you don't care to record/replay: it doesn't matter if
-requests to these hosts succeed or not, and you don't care to manage their
-recorded file.  You can just add those to the ignore list:
+If you don't want requests going to specific server, you can add them to the
+drop list.  For example, Google Analytics, where you don't care that the request
+go through, and you don't want to record it.
 
-```
-Replay.ignore "www.google-analytics.com", "airbrake.io"
+```javascript
+Replay.drop('www.google-analytics.com', 'rollbar.com');
 ```
 
-Likewise, you can tell **node-reply** to allow network access to specific hosts:
+Likewise, you can tell **node-reply** to pass through requests to specific hosts:
 
-```
-Replay.allow "logger.example.com"
+```javascript
+Replay.passThrough('s3.amazonaws.com');
```

Comment:
I am aware that we have calls to `.allow()` in `api-gateway` and that this merge will break things. I'll be preparing a plan for the transition.